packages/admin-ui/src/page/page-header-title.tsx

```diff
@@ -46,6 +46,7 @@ export const PageHeaderTitle = forwardRef(function PageHeaderTitle(
       {onPopNavigation && (
         <Button
```

Comment:
You could solve this on the `Button` itself, right? Define the property `type`, with a default value of `button`. It would solve a lot of use-cases, I suppose. What do you think?